.github/workflows/ci.yml

```diff
@@ -13,6 +13,29 @@ concurrency:
   cancel-in-progress: ${{ github.ref != 'refs/heads/master' }}
 
 jobs:
+  bloop-memory-footprint:
+    timeout-minutes: 120
+    runs-on: ${{ matrix.OS }}
+    strategy:
+      fail-fast: false
+      matrix:
+        OS: ["ubuntu-latest"]
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+        submodules: true
+    - uses: coursier/cache-action@v6.3
+    - uses: coursier/setup-action@v1.1.2
+      with:
+        jvm: 11
+    - name: Build Launcherr
+      run: ./mill -i cli.launcher
+    - name: Build Benchmark
+      run: ./out/cli/launcher/dest/launcher package --assembly gcbenchmark/gcbenchmark.scala
```

Comment:
To avoid hardcode path to launcher, we can use `copyJvmLauncher` from `build.sc`

Reply:
It copies standalone launcher, for which I'm getting:
```
Exception in thread "main" java.lang.NoSuchMethodError: 'java.io.File coursier.paths.CachePath.defaultArchiveCacheDirectory()'
	at coursier.cache.CacheDefaults$.archiveCacheLocation$lzycompute(CacheDefaults.scala:18)
	at coursier.cache.CacheDefaults$.archiveCacheLocation(CacheDefaults.scala:18)
	at coursier.cache.ArchiveCache$.apply(ArchiveCache.scala:141)
	at coursier.jvm.JvmCache$.apply(JvmCache.scala:26)
	at scala.build.options.BuildOptions.javaHomeManager$lzycompute(BuildOptions.scala:240)
	at scala.build.options.BuildOptions.javaHomeManager(BuildOptions.scala:237)
	at scala.cli.commands.SharedOptions.bloopRifleConfig(SharedOptions.scala:189)
	at scala.cli.commands.Package$.run(Package.scala:45)
	at scala.cli.commands.Package$.run(Package.scala:31)
	at caseapp.core.app.CaseApp.main(CaseApp.scala:149)
	at caseapp.core.app.CommandsEntryPoint.main(CommandsEntryPoint.scala:116)
	at scala.cli.ScalaCli$.main0(ScalaCli.scala:173)
	at scala.cli.ScalaCli$.main(ScalaCli.scala:114)
	at scala.cli.ScalaCli.main(ScalaCli.scala)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at coursier.bootstrap.launcher.Bootstrap.main(Bootstrap.java:109)
	at coursier.bootstrap.launcher.ResourcesLauncher.main(ResourcesLauncher.java:79)
```
So I'll keep the hardcoded output path for now